spec/report_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'spec_helper'
+
+describe RepoAudit::Report do
+  let(:repo) { double(Github::Mash, { full_name: 'some-user/not-a-real-repo' }) }
+  let(:report) { described_class.new(repo: repo) }
+  let(:checker) { double(run: nil) }
+
+  before do
+    allow(RepoAudit::LicenseChecker).to receive(:new).with(repo).and_return(checker)
+    allow(ENV).to receive(:fetch).with('GH_TOKEN').and_return('dummy-token')
```

Comment:
Hmmm… I wonder if the token should be passed into `initialize()`. Accessing `ENV` directly feels a little spooky-action-y to me? I'm fine with it for this PR, but might be worth changing later maybe…